pydocx/export/base.py

```diff
@@ -152,7 +152,7 @@ def process_tag_completion(self, result_stack, element, stack):
         return result
 
 
-class DocxParser(MulitMemoizeMixin):
+class PyDocXExporter(MulitMemoizeMixin):
```

Comment:
Nice catch. That typo has been there for a long time. I wonder who might have done that...